internal/controllers/admins.go

```diff
@@ -172,5 +183,17 @@ func (s Server) GetAllUserActivePlans(ctx echo.Context) error {
 		return ctx.JSON(http.StatusInternalServerError, model.ErrorResponse(err.Error(), http.StatusInternalServerError))
 	}
 	return ctx.JSON(http.StatusOK, model.SuccessResponse(plandata, http.StatusOK))
+}
 
+func (s Server) AddUpdatOperation(ctx echo.Context) error {
```

Comment:
There's a typo in this function name.

```suggestion
func (s Server) AddUpdateOperation(ctx echo.Context) error {
```